packages/donations-app/app/components/auth/registrationController.js

```diff
@@ -56,9 +56,7 @@ angular.module('aliceApp')
     };
 
     let extendContactAgreement = function() {
-      let flag = vm.registration.disagreeContact;
-      vm.registration.agreeContact = !flag;
-      vm.registration.agreeAlice = !flag;
+      vm.registration.agreeAlice = vm.registration.agreeContact;
```

Comment:
Why there is no update in agreeContact if the sentence says: " from Alice or the charity."